File: 2.0/java/contacts/getContact.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.contacts.model.Contact;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api login", "api password");
        Contact contact = client.contactsApi().get(463633187003L);
    }
}

File: 2.0/python/contacts/getContact.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.contacts.getContact(
    id=379506003,
    # return only specific fields
    fields='firstName,lastName,workPhone'
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)
